client/src/components/commons/ReviewCard.jsx

```diff
@@ -12,28 +16,85 @@ const {
   read,
   link,
   footer,
+  showPhotoContainer,
+  showPhoto,
+  upVote,
+  downVote,
+  commentsContainer,
+  userPhoto,
+  rightSideContainer,
+  flexShow,
+  commentsTime
 } = styles;
 
-export default function ReviewCard({ username, rating, date, title, location, text, show_id }) {
-  date = new Date(Number(date)).toDateString();
+export default function ReviewCard({ id, username, rating, date, title, location, text, show_id, user_id, user_photo, show_photo, comments }) {
+  const [vote, setVote] = useState(rating)
+  const { currentUser } = useAuth();
+  
+  const onVote = (direction) => {
+    if (direction === 'up') {
+      axios.put(`/api/reviews/${id}/upvote`)
+        .then(setVote(vote + 1))
+        .catch((err) => console.log(err))
+    } else if (direction === 'down') {
+      axios.put(`/api/reviews/${id}/downvote`)
+        .then(setVote(vote - 1))
+        .catch((err) => console.log(err))
+    }
+  }
+
+  date = new Date(Number(date)).toLocaleDateString("en-US");
   return (
     <div className={reviewContainer}>
       <div className={ratingContainer}>
-        <h2>{rating}</h2>
+        {currentUser ? 
+          <Button 
+            onClick={() => {onVote('up')}}>
+            <ExpandLessOutlined id={upVote}/>
+          </Button>
+        :<Button>
+          <ExpandLessOutlined id={upVote}/>
+        </Button>}
+        <h2>{vote}</h2>
+        {currentUser ?  
+          <Button 
+            onClick={() => {onVote('down')}}>
+            <ExpandMoreOutlined id={downVote}/>
+          </Button> 
+        :<Button>
+          <ExpandMoreOutlined id={downVote}/>
+        </Button>}
       </div>
-      <div className={cardContainer}>
-        <div className={header}>
-          <span>{username}</span>
-          <Link to={{ pathname: '/shows', state: show_id }} className={link}>
-            <span>
-              {title}: {location}
-            </span>
-          </Link>
+      <div className={rightSideContainer}>
+        <div className={flexShow}>
+          <div className={showPhotoContainer}>
+            <img src={show_photo} className={showPhoto} alt="show photo"></img>
+          </div>
         </div>
-        <div className={review}>
-          <ReadMore text={text} />
+        <div className={cardContainer}>
+          <div className={header}>
```

Comment:
I'm not really sure what's going on here without seeing it on the page, but if it is getting a classname of 'header', should it maybe be a h tag for semantics sake?